src/qonlinetts.h

```diff
@@ -80,6 +80,30 @@ class QOnlineTts : public QObject
     };
     Q_ENUM(Emotion)
 
+    /**
+     * @brief Defines regional accents to use
+     *
+     * Used only by Google.
+     */
+    enum Region {
+        DefaultRegion,
+        BengaliBangladesh,
+        BengaliIndia,
+        ChineseMandarinChina,
+        EnglishAustralia,
+        EnglishIndia,
+        EnglishUk,
+        EnglishUs,
+        FrenchCanada,
+        FrenchFrance,
+        GermanGermany,
+        PortugueseBrazil,
+        SpanishSpain,
+        SpanishUs,
+        TamilIndia,
+    };
+    Q_ENUM(Region)
```

Comment:
One last thing to consider. What if we use [QLocale countries](https://doc.qt.io/qt-5/qlocale.html#Country-enum) instead of our own enum? So `setRegions` will accept `QOnlineTranslator::Language` and `QLocale::Country`. What do you think?

I created `QOnlineTranslator::Language` enum only because [QLocale languages](https://doc.qt.io/qt-5/qlocale.html#Language-enum) didn't have all languages.